include/maidsafe/routing/node_info.h

```diff
@@ -34,43 +34,36 @@ namespace routing {
 struct NodeInfo {
   using PublicPmid = passport::PublicPmid;
 
-  NodeInfo(NodeId id, PublicPmid dht_fob) : id(id), dht_fob(std::move(dht_fob)) {}
-  NodeInfo() = default;
+  NodeInfo(NodeId id_in, PublicPmid dht_fob_in, bool connected_in)
+      : id(id_in), dht_fob(std::move(dht_fob_in)), connected(connected_in) {}
+
+  NodeInfo() : id(), dht_fob(), connected(false) {}
+
   NodeInfo(const NodeInfo&) = default;
 
   NodeInfo(NodeInfo&& other) MAIDSAFE_NOEXCEPT : id(std::move(other.id)),
-                                                 dht_fob(std::move(other.dht_fob)) {}
+                                                 dht_fob(std::move(other.dht_fob)),
+                                                 connected(std::move(other.connected)) {}
+
   NodeInfo& operator=(const NodeInfo&) = default;
 
   NodeInfo& operator=(NodeInfo&& other) MAIDSAFE_NOEXCEPT {
     id = std::move(other.id);
     dht_fob = std::move(other.dht_fob);
+    connected = std::move(other.connected);
```

Comment:
Minor: there is no point in moving a boolean as it does the same as just copying it. An argument against it is that it might be confusing and it obfuscates the code.